src/components/ModuleCard/styles.module.css

```
@@ -0,0 +1,57 @@
+}
+
+.body {
+  padding: 0.75rem 1rem 1rem;
+  flex: 1;
+}
+
+.title {
+  font-size: 0.85rem;
+  font-weight: 600;
+  margin-bottom: 0.4rem;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)